proxy/src/main/java/com/velocitypowered/proxy/ProxyOptions.java

```diff
@@ -75,4 +93,40 @@ boolean isHelp() {
   public @Nullable Boolean isHaproxy() {
     return this.haproxy;
   }
+
+  public boolean isIgnoreConfigServers() {
+    return this.ignoreConfigServers;
+  }
+
+  public List<ServerInfo> getServers() {
+    return this.servers;
+  }
+
+  private static class ServerInfoConverter implements ValueConverter<ServerInfo> {
+
+    @Override
+    public ServerInfo convert(String s) {
+      String[] split = s.split(":", 3);
+      if (split.length < 2 || split.length > 4) {
+        throw new ValueConversionException("Invalid server format. Use <name>:<address>");
+      }
+      InetAddress address;
+      try {
+        address = InetAddress.getByName(split[1]);
+      } catch (UnknownHostException e) {
+        throw new ValueConversionException("Invalid hostname for server flag with name: " + split[1]);
+      }
```

Comment:
Use `AddressUtil.parseAddress()` for consistency with the `velocity.toml` reader. (Long term, I'd love to make Velocity use a YAML file and allow at least the proxy itself to be a 12-factor app, but I lack the time.)

Reply:
Have swapped over :)